src/editors/playgroundController.ts

```diff
@@ -412,18 +395,26 @@ export default class PlaygroundController {
       }
     }
 
+    this._outputChannel.clear();
+
     const evaluateResponse: ExecuteAllResult = await this.evaluateWithCancelModal();
 
-    this._outputChannel.clear();
-    if (evaluateResponse.outputLines) {
+    if (
+      evaluateResponse &&
+      evaluateResponse.outputLines &&
+      evaluateResponse.outputLines.length > 0
+    ) {
```

Comment:
You should be able to do something like 
```suggestion
    if (evaluateResponse?.outputLines?.length) {
```
here :)